lib/src/10_auth/auth_screen.dart

```diff
@@ -157,27 +184,37 @@ class _FauiAuthScreenState extends State<FauiAuthScreen> {
     };
 
     return Column(children: <Widget>[
-      TextField(
-        autofocus: true,
-        controller: emailController,
-        decoration: InputDecoration(
-          labelText: resolvePhrase(FauiPhrases.EmailTextField,"Email"),
+      Focus(
+        child: TextField(
+           autofocus: true,
+           controller: emailController,
```

Comment:
Can we wrap "Focus(...)" into function?

Reply:
Done. The function is callled " _buildTextBox"